src/Tribe/RSVP.php

```diff
@@ -736,11 +753,18 @@ public function send_non_attendance_confirmation( $order_id, $event_id ) {
 			return;
 		}
 
-		$headers     = apply_filters( 'tribe_rsvp_email_headers', array( 'Content-type: text/html' ) );
-		$attachments = apply_filters( 'tribe_rsvp_email_attachments', array() );
-		$to          = apply_filters( 'tribe_rsvp_email_recipient', $to );
+		/**
+		 * Filters for RSVP non attendance confirmation email
+		 * @since TBD added new parameters $event_id and $order_id to the filters below
+		 */
```

Comment:
Here also: ideally we'll have a docblock per filter. If not, though, change it so it documents the first filter only (because that's what the doc parser will associate it with).